keyboards/niu_mini/readme.md

```diff
@@ -13,4 +12,4 @@ Make example for this keyboard (after setting up your build environment):
 
     make niu_mini:default:dfu
```

Comment:
Should this not be just `make niu_mini:default`?

The PS2AVRGB template has a separate flashing example block, but not AVR, hmm.